ops/testing.py

```diff
@@ -2254,7 +2254,17 @@ def status_get(self, *, is_app: bool = False):
         else:
             return self._unit_status
 
-    def status_set(self, status: '_StatusName', message: str = '', *, is_app: bool = False):
+    def status_set(
```

Comment:
Per above, let's revert the new argument (and revert the formatting change to avoid additional diff).

Reply:
reverted w/ changes, thank you!